00:00:00.001361 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.001365 main     Log opened 2020-11-23T14:18:36.136758900Z
00:00:00.001365 main     Build Type: release
00:00:00.001368 main     OS Product: Windows 10
00:00:00.001369 main     OS Release: 10.0.18363
00:00:00.001370 main     OS Service Pack: 
00:00:00.093720 main     DMI Product Name: 20L6S9U200
00:00:00.100628 main     DMI Product Version: ThinkPad T480
00:00:00.100660 main     Firmware type: BIOS
00:00:00.100676 main     Host RAM: 32426MB (31.6GB) total, 23725MB (23.1GB) available
00:00:00.100680 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.100680 main     Process ID: 7804
00:00:00.100681 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.164777          VirtualBox: object creation starts
00:00:00.165898          Home directory: 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox'
00:00:00.165999          Installed Drivers:
00:00:00.169617            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.169956            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.170236            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.170641            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.173383          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.175632          usbLibGetDevices: Starting USB device enumeration
00:00:00.177102          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.177140          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.177176          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.177209          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.178160          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.178303          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 2
00:00:00.178333          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.179596          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.179666          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.179676          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.179687          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.179706          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.179713          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0ac9)
00:00:00.179729          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0aca)
00:00:00.257047          HostDnsMonitor: initializing
00:00:00.264911          HostDnsMonitor: updating information
00:00:00.264932          HostDnsMonitor: old information
00:00:00.264933            no server entries
00:00:00.264935            no domain set
00:00:00.264935            no search string entries
00:00:00.264935          HostDnsMonitor: new information
00:00:00.264936            server 1: 127.0.0.1
00:00:00.264936            domain: ni.corp.natinst.com
00:00:00.264937            no search string entries
00:00:00.362377          VD: VDInit finished with VINF_SUCCESS
00:00:00.363797          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.365029          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.374858          OCI: Local config file 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.375046          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.375047          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.375055          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.375077          VirtualBox: object created
00:00:00.886759          Launched VM: 71221696 pid: 15232 (0x3b80) frontend: GUI/Qt name: LinuxLive
00:00:04.525761          Load [C:\Program Files\Oracle\VirtualBox\ExtensionPacks\Oracle_VM_VirtualBox_Extension_Pack\win.amd64\VBoxHostWebcam.DLL] vrc=VINF_SUCCESS
00:00:04.761144          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:05.023358          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:05.138420          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:05.217121          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:05.299025          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:05.377982          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
01:03:51.890957          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
01:03:52.561671          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
01:03:58.158718 main     VirtualBox: object deletion starts
01:03:58.513686 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
01:03:58.520329 main     HostDnsMonitor: shutting down ...
01:03:58.525768 main     HostDnsMonitor: shut down
01:03:58.534143 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
01:03:58.539374 main     VirtualBox: object deleted
